### PR TITLE
feat: add NumberExtension to create SizedBox from num

### PR DESCRIPTION
**Pull request by jasimameen**
Usage

``` dart
// regular
SizedBox(height: 20);

// by extension
20.hSizedBox; // same output as above
```

### CODE REVIEW

File: lib/src/super_extensions_base.dart [hunk 1]
```diff
@@ -83,3 +83,13 @@ extension StringExtension on String {
           .map((str) => str.capitaliseFirstChar())
           .join(' ');
 }
+
+extension NumberExtension on num {
+  /// creates a Empty [SizedBox] with height as the given number
+  /// 
+  SizedBox get pH => SizedBox(height: toDouble());
```
Comment (AbhishekDoshi26) [inline lib/src/super_extensions_base.dart]: I would suggest, having a name as `pH` can be a bit confusing. For example, if I don't open the extension file, I won't understand that pH creates a SizedBox with height. Maybe having name like `hSizedBox` or something else that makes more sense?

Comment (AbhishekDoshi26) [reply lib/src/super_extensions_base.dart]: Also, please update the `Readme.md` file too to include the new extension methods.

**Review by AbhishekDoshi26 — CHANGES_REQUESTED** [review]

File: lib/src/super_extensions_base.dart (hunk 1 above)
Comment (jasimameen) [reply lib/src/super_extensions_base.dart]: Yes, That makes sense. I'll update it.

File: lib/src/super_extensions_base.dart [hunk 2]
```diff
@@ -83,3 +83,13 @@ extension StringExtension on String {
           .map((str) => str.capitaliseFirstChar())
           .join(' ');
 }
+
+extension NumberExtension on num {
+  /// creates a Empty [SizedBox] with height as the given number
+  /// 
+  SizedBox get hSizedBox => SizedBox(height: toDouble());
+
+  /// creates a Empty [SizedBox] with width as the given number
```
Comment (AbhishekDoshi26) [inline lib/src/super_extensions_base.dart]: here it should be `Creates an empty SizedBox with [width] as the given number.
same for height.

**Review by AbhishekDoshi26 — CHANGES_REQUESTED** [review]

File: lib/src/super_extensions_base.dart (hunk 2 above)
Comment (jasimameen) [reply lib/src/super_extensions_base.dart]: sure, updated

**Review by AbhishekDoshi26 — APPROVED** [review]

**Comment by AbhishekDoshi26** [comment]
LGTM! Thanks. Merging 🚀

**Pull request merged**